street-comics-api/app/controllers/api/v1/characters_controller.rb

```diff
@@ -12,7 +12,13 @@ def index
 
         client_params = params
                           .with_defaults(default_params)
-                          .permit(:offset, :limit, :order_by_field, :order_by_direction)
+                          .permit(
```

Comment:
This filtering is becoming redundant now that the adapter layer only maps permitted fields anyway.

Will consider changing that in future refactorings.